wallet/lib/components/maintenance_mode_widgets.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+import '../generated/locale_keys.g.dart';
+import '../utils/constants.dart';
```

Comment:
Use absolute path instead of relative one.

Reply:
Good suggestion i will add a linter for this.

Reply:
Thank you.